src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
In the future, I might try to make the tough call to extract out keyboard controls. It's not really great to have for first-person VR and we should encourage discovering better locomotion mechanisms. WASD has been a bit of a crutch.

For third-person VR is fine, but then WASD + third-person is too specific of a use-case to include in core.

@dmarcos Should we just move it out now?